tests/test_handler.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+import os
+from datetime import datetime, timedelta
+
+from RadClass.analysis import RadClass
+from tests.create_file import create_file
+
+
+def test_integration():
+    filename = 'testfile.h5'
+    datapath = '/uppergroup/lowergroup/'
+    labels = {'live': '2x4x16LiveTimes',
+              'timestamps': '2x4x16Times',
+              'spectra': '2x4x16Spectra'}
+
+    start_date = datetime(2019, 2, 2)
+    delta = timedelta(seconds=1)
+    timestamps = np.array([])
+    for i in range(1000):
+        timestamps = np.append(timestamps, start_date.timestamp())
```

Comment:
Best to pre-allocate this and then assign the values rather than append values in each case.

Reply:
Further, can't you use `np.arange()` or something to just create this vector as you need it?

Reply:
✔️ This form seems to work:
```
timestamps = np.arange(start_date, start_date + (timesteps * delta), delta).astype('datetime64[s]').astype('float64')
```